### PR TITLE
Update utils to latest stable

### PR DESCRIPTION
**Pull request by graysky2**
* autoconf from 2.69 to 2.71
* isl from 0.22 to 0.24

Signed-off-by: John Audia <graysky@archlinux.us>

### CODE REVIEW

File: packages/autoconf/2.71/0001-Update-autoconf-and-add-new-version.patch [hunk 1]
```diff
@@ -0,0 +1,35 @@
+From a09e46a5413066896e7a18c0032b5e2da35df1f9 Mon Sep 17 00:00:00 2001
```
Comment (cpackham) [inline packages/autoconf/2.71/0001-Update-autoconf-and-add-new-version.patch:1]: This file seems to have been added in error

**Review by cpackham — CHANGES_REQUESTED** [review]
I'd also prefer real names in the Sign-off line.

File: packages/autoconf/2.71/0001-Update-autoconf-and-add-new-version.patch (hunk 1 above)
Comment (graysky2) [reply packages/autoconf/2.71/0001-Update-autoconf-and-add-new-version.patch]: Fixed both.

**Comment by cpackham** [comment]
Looks better thanks

**Review by cpackham — APPROVED** [review]

**Pull request merged**